### DISCUSSION TITLE
Table layouting with rows and columns DnD (Full interactive design)

### DISCUSSION THREAD

**Discussion by hamzalshanti** [xyflow/xyflow#3394] [Q&A]
Hey there

is attached picture it doable by react flow? I have table view that have nodes inside cell, nodes with cells can be connectable and nodes for diffrent cells can be connectable together, also Columns and rows can be sortable using DnD.

![reactflow](https://github.com/wbkd/react-flow/assets/66948735/0c4ed36e-54db-4205-a0f1-d543b35f695e)

**Comment by hamzalshanti** [comment]
@moklick

**Reply by moklick** [reply]
Seems like a very complicated thing to do but it could be possible somehow. I would try to implement a very basic MVP to see if it's possible. You can specify an extent for each node which would be the parent cell. Can you connect nodes between different columns? If not, you could create a separate flow for each column.

**Reply by hamzalshanti** [reply]
@moklick Thank you for your response. Our business requirement necessitates connecting nodes across various columns. I initially considered implementing separate flows; however, this approach is not aligned with our business requirements. Additionally, I can offer an MVP that closely resembles the provided image, albeit without the Drag and Drop (DnD) functionality, which may be of assistance.

**Comment by Kennn-dev** [comment]
I have a feature like this too , stuck when implement DnD in reactflow
